backend/.env.example

```
@@ -54,7 +45,9 @@ AWS_SECRET_ACCESS_KEY=
 AWS_REGION=us-east-1
 AWS_S3_BUCKET_NAME=AWS_S3_BUCKET
 
-# Sender Email Configuration
-SENDER_EMAIL="example@email.com"
-# (NOT A REAL KEY, REPLACE WITH YOUR OWN)
-SENDGRID_KEY="SG.0a1b2c3d4e5f6g7h8i9j0k1l2m3n4o5p6q7r8s9t0u1v2w3x4y5z"
+# Emails
+SMTP_SERVER="smtp.gmail.com"
+SMTP_PORT=465
+SMTP_USER="example@email.com"
+SMTP_PASSWORD="your-email-password"
```

```suggestion
SMTP_PASSWORD="changethis"
```
please use `changethis` or something that contains `example` in it, to be sure our validations catch it and throw warnings.